assets/i18n/en.json

```diff
@@ -296,6 +296,13 @@
     "I18N_PLAYER_FEEDBACK_TOOLTIP": "Feedback",
     "I18N_PLAYER_FORWARD": "Forward",
     "I18N_PLAYER_GO_TO_LIBRARY": "Go to library",
+    "I18N_PLAYER_HINT": "Hint",
+    "I18N_PLAYER_HINTS_EXHAUSTED": "Sorry, I'm out of hints!",
+    "I18N_PLAYER_HINT_IS_USABLE": "Click here for a hint!",
+    "I18N_PLAYER_HINT_NOT_USABLE": "Try thinking a bit more before asking for another hint!",
```

Comment:
This is a minor gripe, but this text doesn't make sense on the first hint, since it's not "another" hint, it's just "a" hint.